base_custom_attribute/tests/test_build_view.py

```diff
@@ -93,46 +114,47 @@ def _check_attrset_visiblility(self, attrs, set_ids):
         )
 
     def _get_attr_element(self, name):
-        views = self.env["res.partner"]._build_attribute_view()
-        return views.find("group/field[@name='{}']".format(name))
+        eview = self.env["res.partner"]._insert_attribute(deepcopy(self.eview))
+        return eview.xpath("//field[@name='{}']".format(name))[0]
 
+    # TESTS on CUSTOM ATTRIBUTES
     def test_group_order(self):
-        views = self.env["res.partner"]._build_attribute_view()
-        groups = [g.get("string") for g in views.getchildren()]
+        eview = self.env["res.partner"]._insert_attribute(deepcopy(self.eview))
```

Comment:
do not change this test.
They test the build_attribute_view.
instead just a one or two test for testing the insert method